src/pages/lessons/[slug].tsx

```diff
@@ -99,16 +99,18 @@ type MetadataProps = {
     },
   ]
   summary: string
+  [cssRelated: string]: any
 }
 
 const Metadata: FunctionComponent<MetadataProps> = ({
   title,
   instructor,
   tags,
   summary,
+  ...restProps
 }: MetadataProps) => {
   return (
-    <div>
+    <div {...restProps}>
```

Comment:
Don’t know if this applies here but it was interesting https://twitter.com/markdalgleish/status/1305714449475489792?s=20